src/modules/MediaKit/RingBuffer.cpp

```diff
@@ -0,0 +1,129 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include "RingBuffer.hpp"
+
+RingBuffer::RingBuffer( int size )
+{
+	 initialized = false;
```

Comment:
Is it possible to change all indentations to spaces?

Reply:
Yes. Feel free to change to fit your coding style.